src/components/button/button.js

```diff
@@ -73,7 +89,22 @@ class Button extends React.Component {
      * @type {String}
      * @default medium
      */
-    size: PropTypes.string
+    size: PropTypes.string,
+
+    /**
+     * Sets a second bit of text under the main text, feinter and smaller.
```

Comment:
They use feint to describe extremely lightly lined paper and I extrapolated from that, however, I might actually be wrong to use the word like this.